<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zhu_JIaMian</class>
 <widget class="QWidget" name="Zhu_JIaMian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1174</width>
    <height>748</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1174</width>
    <height>748</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1181</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>实时数据</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>691</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/TuPian_ZiYuan/JiaoShiYuan_ShiJiao2.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>651</width>
       <height>281</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2" rowspan="2">
       <widget class="QLabel" name="lab_TiWen2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QLabel" name="lab_XinLv1">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>心率：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QLabel" name="lab_WeiZhi1">
        <property name="text">
         <string>位置：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="2">
       <widget class="QLabel" name="lab_XueYang2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="lab_RiQi1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>日期：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QLabel" name="lab_XueYang1">
        <property name="text">
         <string>血氧：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QLabel" name="lab_RiQi2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="2">
       <widget class="QLabel" name="lab_XinLv2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" rowspan="2">
       <widget class="QLabel" name="lab_WeiZhi2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="lab_CheZhu1">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>车主：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="2">
       <widget class="QLabel" name="lab_TiWen1">
        <property name="text">
         <string>体温：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="lab_CheZhu2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lab_TiShi1">
        <property name="text">
         <string>   提示：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="lab_TiShi2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_ShuaXin">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>630</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>违规记录</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1171</width>
       <height>691</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/TuPian_ZiYuan/WeiGui_JiLu.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>900</width>
       <height>591</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
